src/commands/settle.js

```diff
@@ -26,21 +26,22 @@ module.exports = {
         yargs.coerce('amount', arg => arg);
     },
     handler: async (argv) => {
-        const { amount, currency, gas, price } = argv;
+        const { currency } = argv;
         const config = require('../config');
+        
         const provider = await nahmii.NahmiiProvider.from(config.apiRoot, config.appId, config.appSecret);
+        const tokenInfo = await provider.getTokenInfo(currency);
```

Comment:
This async call is now done outside of the try...catch scope => unhandled rejection if it fails